.config/nextest.toml

```diff
@@ -1,2 +1,6 @@
 [profile.default]
 slow-timeout = { period = "60s", terminate-after = 3 }
+
+[[profile.default.overrides]]
+filter = 'package(/^proxy/) & (test(/^scram::threadpool::/) | test(/^auth::backend::tests::auth_quirks/))'
+threads-required = 2
```

Comment:
Is this still required? Could use an explaining comment.

Reply:
I think they can be removed. There's only one active thread at a time in these tests